app/models/manageiq/providers/automation_manager.rb

```diff
@@ -11,7 +11,8 @@ class ManageIQ::Providers::AutomationManager < ManageIQ::Providers::BaseManager
 
   has_many :configured_systems,           :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_profiles,       :dependent => :destroy, :foreign_key => "manager_id"
-  has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
+  has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id",
+                                          :class_name => "::ConfigurationScript"
```

Comment:
@Fryguy @blomquisg @gmcculloug @bzwei 
This would get back both job_templates and workflows, but I don't like that it's breaching into the base class space.

Reply:
you don't need the class_name bit, because it's implied.

Reply:
That is exactly how we'd expected it to work from the original discussion...not sure I understand your concern.

Reply:
Without the `class_name`, it will default to the `AutomationManager::ConfigurationScript` (and that gives only the `job_templates`)

Reply:
@Fryguy 
Our comments are interwinding and so let me try to clarify again:
 1. We need to pin the `class_name` to the base class in order to get what we want (get back both job_templates/workflows)
2. Pinning the relationship (the `class_name`) from a subclass level object to a base class level object (i.e. `ManageIQ::Providers::AutomationManager` vs `::ConfigurationScript) is kind of breaching the abstraction layers. It's not the end of the world. But why don't we simply move the relationship to the 2 base classes (`EMS` vs `ConfigurationScript`) ?